pkg/platform/runtime/runtime.go

```diff
@@ -66,9 +68,10 @@ func newRuntime(target setup.Targeter, an analytics.Dispatcher, svcModel *model.
 }
 
 // New attempts to create a new runtime from local storage.  If it fails with a NeedsUpdateError, Update() needs to be called to update the locally stored runtime.
-func New(target setup.Targeter, an analytics.Dispatcher, svcm *model.SvcModel) (*Runtime, error) {
+func New(target setup.Targeter, an analytics.Dispatcher, svcm *model.SvcModel, out output.Outputer) (*Runtime, error) {
 	if strings.ToLower(os.Getenv(constants.DisableRuntime)) == "true" {
-		return &Runtime{disabled: true, target: target}, nil
+		out.Notice(locale.Tl("notice_runtime_disabled", "Skipping runtime setup because it was disabled by an environment variable"))
```

Comment:
Can we use `fmt.Fprintf(os.Stderr, ..)` here? It's been a conscious decision to keep the outputer out of the runtime mechanic. This is situation obviously calls for it, but seeing as the use-case is internal I think it's acceptable to use `fmt` here.

I worry that if we pass the outputer in that we will start using it elsewhere and break the original design goals (which is to use the event handles to handle output).